python/PiFinder/camera_debug.py

```diff
@@ -40,7 +42,8 @@ def capture(self) -> Image.Image:
         sleep_time = self.exposure_time / 1000000
         time.sleep(sleep_time)
         logging.debug("CameraDebug exposed for %s seconds", sleep_time)
-        return self.image
+        self.image_bool = not self.image_bool
+        return self.image if self.image_bool else self.image2
```

Comment:
This is a very nice addition.  Eventually we might want to expand this even more to just cycle through all images in dir :+1: